sdk/communication/Azure.Communication.Chat/src/Models/ChatModelFactory.cs

```diff
@@ -4,13 +4,15 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using Azure.Core;
 
 namespace Azure.Communication.Chat
 {
     /// <summary>
     /// Model factory that enables mocking for the Chat library.
     /// </summary>
-    public static class ChatModelFactory
+    [CodeGenType("Azure.Communication.Chat.AzureCommunicationServicesModelFactory")]
+    public static partial class ChatModelFactory
```

Comment:
Is it easier to change the `title` to `Chat` in autorest.md?

Reply:
Fixed